### PR TITLE
Add documentation for Stackdriver Monitoring in the README.

### PR DESCRIPTION
**Pull request by supriyagarg**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -268,6 +270,44 @@ to Stackdriver Logging using this Client Library.
...
+    from gcloud import monitoring
+    client = monitoring.Client()
+    for descriptor in client.list_metric_descriptors():
+        print(descriptor.type)
+
+Display CPU utilization across your GCE instances during the last five minutes:
+
+.. code:: python
+
+    METRIC = 'compute.googleapis.com/instance/cpu/utilization'
```
Comment (dhermes) [inline README.rst]: S/`METRIC`/`metric`

**Comment by dhermes** [comment]
LGTM other than the constant var name for `METRIC`. Ping me after the change and I'll merge. Please squash the change into a single commit if it isn't too big a hassle (you can just change the file and then run a bare `git commit --amend` to fold into the original commit and then `git push --force` to overwrite the original commit)

**Comment by dhermes** [comment]
Ah I just realized there are two commits already. Can you squash all into a single commit? (Sorry, doing code review from my phone)

**Comment by supriyagarg** [comment]
Thanks. I made the edit and squashed the commits into one.

**Pull request merged**